docs/developer/README.md

```diff
@@ -22,6 +22,18 @@ Visit the running application at [http://localhost:8080](http://localhost:8080).
 
 We use the branch convention of `initials/branch-topic` (ex: `lmm/fix-footer`). This allows for automated deployment to a developer sandbox namespaced to the initials.
 
+## Merging and PRs
+
+To bring your feature branch up-to-date wih main:
```

Comment:
Want to add a note about merge vs rebase and link to the article we discussed in Slack as background?

Reply:
I just saw this note. Absolutely, I'll create another PR.